### DISCUSSION TITLE
Can't connect using domain name

### DISCUSSION THREAD

**Discussion by jordy33** [jamulussoftware/jamulus#975] [Technical Support & Help]
**Describe the bug**
I have a private server. When i give the ip address "189.232.94.215" to the people that connects to the server every thing works fine and every body can connect.  But when i give the domain name: jorge33.ddns.net no body can connect.

**To Reproduce**
Use de domain name:  jorge33.ddns.net

**Expected behavior**

**Screenshots**

**Operating system**
Client: Mac OS  
Server: Ubuntu

**Version of Jamulus**
3.6.2
**Additional context**

**Comment by hoffie** [comment]
Hmm, any chance you can get a network trace while the problem appears?
The name seems to resolve properly (although a bit slow). I tried to connect and it did not work for me, but it did not work for me using the IP directly either.
Is your server running right now?

**Comment by chrisrimple** [comment]
Is this a cloud or local server?  Are you sure that you have ddns.net configured correctly?  The fact that users can connect to the direct IP implies that there are no port forwarding, etc. issues, so it seems like the DNS mapping is the issue.

**Comment by jordy33** [comment]
> Hmm, any chance you can get a network trace while the problem appears?
> The name seems to resolve properly (although a bit slow). I tried to connect and it did not work for me, but it did not work for me using the IP directly either.
> Is your server running right now?

Nope is down... I'll turn on.

**Comment by hoffie** [comment]
> Nope is down... I'll turn on.

Is it still up? I still cannot connect, even using the IP.

When you or your colleague tested based on the IP, where was this person located? Were they in a different location with a different ISP/dialup/cable connection or did they share it with you (= same network)?

**Comment by ann0see** [comment]
I've now converted it to a discussion since I think it's not a bug in Jamulus, rather a configuration/network issue.
